### PR TITLE
postmaster - Schemes field updates

### PR DESCRIPTION
**Pull request by rsdunya**
The edit channel page should now provide a dropdown that allows you to also update the scheme.

1. Navigate to: channels/types/postmaster/claim
2. Create a postmaster channel. 
3. Check the postgres database channels_channel table to ensure that the newly created channels schema is properly set: `select id, channel_type, schemes  from channels_channel where channel_type = 'PSM' and is_active is true;`
4. Now navigate to the channel's update page ex (channels/channel/update/[channel_id]/).
Screen Shot 2020-03-05 at 12 31 34 PM

Screen Shot 2020-03-05 at 12 31 30 PM

5. Modify the scheme and save the changes.
6. Verify in the postgres channels_channel table that the channels scheme has been updated: `select id, channel_type, schemes  from channels_channel where channel_type = 'PSM' and is_active is true;`

### CODE REVIEW

File: temba/channels/types/postmaster/views.py [hunk 1]
```diff
@@ -25,7 +26,6 @@ class ClaimView(BaseClaimNumberMixin, SmartFormView):
     uuid = None
 
     class Form(ClaimViewMixin.Form):
-
         CHAT_MODE_CHOICES = (("WA", _("WhatsApp")), ("TG", _("Telegram")), ("SMS", _("SMS")))
```
Comment (andrewkcarter) [inline temba/channels/types/postmaster/views.py]: Can this be externalized and defined in the settings file? Then we can expand modes very easily as we add them to Postmaster.

Comment (rsdunya) [reply temba/channels/types/postmaster/views.py]: @andrewkcarter Yes it can be

**Comment by rsdunya** [comment]
> Externalize the Mode options in the settings file. That application should not break if the setting is missing, but simply default to an empty list. Add the values to the settings files in rapidpro-docker.

Heres the supplementary PR: https://github.com/istresearch/rapidpro-docker/pull/45

**Comment by andrewkcarter** [comment]
Code reviewed, still need to run a test.

**Pull request merged**